lib/ConsumerImpl.h

```diff
@@ -295,6 +300,12 @@ class ConsumerImpl : public ConsumerImplBase {
     MapCache<std::string, ChunkedMessageCtx> chunkedMessageCache_;
     mutable std::mutex chunkProcessMutex_;
 
+    const long expireTimeOfIncompleteChunkedMessageMs_;
+    DeadlineTimerPtr checkExpiredChunkedTimer_;
+    std::atomic_bool expireChunkMessageTaskScheduled_ = {false};
```

Comment:
You can simply use the brace initialization for those have no copy constructors.

```suggestion
    std::atomic_bool expireChunkMessageTaskScheduled_{false};
```